src/shogun/mathematics/NormalDistribution.cpp

```diff
@@ -0,0 +1,97 @@
+#include <shogun/io/SGIO.h>
+#include <shogun/mathematics/NormalDistribution.h>
+
+#include <cmath>
+#include <limits>
+
+using namespace shogun;
+
+NormalDistribution::NormalDistribution(float64_t mean, float64_t stddev)
+{
+	m_mean = mean;
+	m_stddev = stddev;
+
+	/** init ziggurat variables */
+	m_blockCount = 128;
```

Comment:
all of these could be constexpr no?

Reply:
constexpr std::array is very hard to initialize